charts/observability-pipeline/templates/control-plane-deployment.yaml

```diff
@@ -25,11 +25,15 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.controlPlane.image.repository }}:{{ .Values.controlPlane.image.tag }}"
           imagePullPolicy: {{ .Values.controlPlane.image.pullPolicy }}
-          {{- if .Values.controlPlane.telemetry.enabled }}
           args:
+          {{- if .Values.controlPlane.telemetry.enabled }}
```

Comment:
if neither of these conditions are true then we'll render an empty `args` which I am pretty sure is an issue.

Reply:
I added a condition to wrap the whole `args` section